### PR TITLE
Restored lost preparation section that was dropped in the Gatsby migration

### PR DESCRIPTION
**Pull request by rhetoric101**
### Tell us why

The Drupal to Gatsby migration left out the preparation section of the Kafka integration, which I just restored.

### Anything else you'd like to share?

This PR is related to GitHub issue https://github.com/newrelic/docs-website/issues/1123.

The review just needs to compare the content in this PR with the content here:

https://docs-dev.newrelic.com/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration#prepare-for-install

### CODE REVIEW

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx [hunk 1]
```diff
@@ -29,6 +29,69 @@ Before installing the integration, make sure that you meet the following require
...
+
+#### Bootstrap [#bootstrap]
+
+With the bootstrap mechanism, the integration uses a bootstrap broker to perform the autodiscovery. This is a broker whose address is well known and that will be asked for any other brokers it is aware of. The integration needs to be able to contact this broker in the address provided in the bootstrap_broker_host parameter for bootstrap discovery to work.
+
+#### Zookeeper [#zookeeper]
+
+Alternatively, the Kafta integration can also talk to a Zookeeper server in order to obtain the list of brokers. To do this, the integration needs to be provided with the following:
+
+The list of Zookeeper hosts to contact (zookeeper_hosts).
```
Comment (urbiz-nr) [inline src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx:48]: This is a bullet in the original doc.

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx [hunk 2]
```diff
@@ -29,6 +29,69 @@ Before installing the integration, make sure that you meet the following require
...
+The proper authentication secrets to connect with the hosts.
+Together with the list of brokers it knows about, Zookeeper will also advertise which connection mechanisms are supported by each broker.
```
Comment (urbiz-nr) [inline src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx:50]: This is a bullet in the original doc.

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx [hunk 3]
```diff
@@ -29,6 +29,69 @@ Before installing the integration, make sure that you meet the following require
...
+#### Bootstrap [#bootstrap]
+
+With the bootstrap mechanism, the integration uses a bootstrap broker to perform the autodiscovery. This is a broker whose address is well known and that will be asked for any other brokers it is aware of. The integration needs to be able to contact this broker in the address provided in the bootstrap_broker_host parameter for bootstrap discovery to work.
+
+#### Zookeeper [#zookeeper]
+
+Alternatively, the Kafta integration can also talk to a Zookeeper server in order to obtain the list of brokers. To do this, the integration needs to be provided with the following:
+
+The list of Zookeeper hosts to contact (zookeeper_hosts).
+The proper authentication secrets to connect with the hosts.
```
Comment (urbiz-nr) [inline src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx:49]: This is a bullet in the original doc.

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx (hunk 2 above)
Comment (urbiz-nr) [inline src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx:50]: This is a separate line in the original doc.

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx [hunk 4]
```diff
@@ -29,6 +29,69 @@ Before installing the integration, make sure that you meet the following require
...
+    *Hosts defined in bootstrap_broker_host over the Kafka protocol, using the Kafka broker’s authentication/transport mechanisms (if autodiscover_strategy is set to bootstrap).
+    * All brokers in the cluster over the Kafka protocol and port, using the Kafka brokers' authentication/transport mechanisms.
+    * All brokers in the cluster over the JMX protocol and port, using the authentication/transport mechanisms specified in the JMX configuration of the brokers.
+    * All producers/consumers specified in producers and consumers over the JMX protocol and port, if you want producer/consumer monitoring. JMX settings for the consumer must be the same as for the brokers.
+    
+<Callout variant="important">
+
+For the cloud: By default, Security Groups (and their equivalents in other cloud providers) in AWS do not have the required ports open by default. JMX requires two ports in order to work: the JMX port and the RMI port. These can be set to the same value when configuring the JVM to enable JMX and must be open for the integration to be able to connect to and collect metrics from brokers.
```
Comment (urbiz-nr) [inline src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx:92]: "For the cloud" and "must" in bold in the original doc.

**Comment by urbiz-nr** [comment]
@rhetoric101 - I've added a couple of comments. You can go ahead and merge once you review them.

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx (hunk 3 above)
Comment (rhetoric101) [reply src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx]: @urbiz-nr, I converted this to a bullet!

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx:51]: @urbiz-nr, I converted this to a bullet!

File: src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx (hunk 4 above)
Comment (rhetoric101) [reply src/content/docs/integrations/host-integrations/host-integrations-list/kafka-monitoring-integration.mdx]: @urbiz-nr, I added the bold elements and also noticed I had missed some code formatting; so, I added that, too. Thanks so much!

**Pull request merged**